src/app/core/products/merchandise-api.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@angular/core";
+import { Observable, of } from "rxjs";
+import { map } from "rxjs/operators";
+
+import { Merchandise } from "@shared/models/classes/merchandise.class";
+
+@Injectable({
+    providedIn: "root"
+})
+export class MerchandiseApiService {
+    merchandise: Merchandise[] = [
+        new Merchandise({
+            id: "7",
+            name: "Dak Tote Bag",
+            price: "14.00",
+            collection: undefined,
+            description: "Premium quality: 300 gr./m2",
+            slug: "tote",
+            dimensions: "41 x 42 cm",
+            material: "Cotton",
+        })
+    ];
+
+    constructor() {}
+
+    getMerchandise(): Observable<Merchandise[]> {
```

Comment:
best practice (as far as I have seen it so far...) is to use getAll{resource} for the GET / route and to use getOne{resource} for the get /:id or /:slug

Just a quick note, we will fix when we change to the actual api so need to fix in this ticket